libr/core/cconfig.c

```diff
@@ -3147,6 +3147,7 @@ R_API int r_core_config_init(RCore *core) {
 	SETBPREF ("asm.cmt.patch", "false", "Show patch comments in disasm");
 	SETBPREF ("asm.cmt.off", "nodup", "Show offset comment in disasm (true, false, nodup)");
 	SETBPREF ("asm.payloads", "false", "Show payload bytes in disasm");
+	SETBPREF ("asm.followcf.hide", "false", "Hide call/jump/return instructions (pde)");
```

Comment:
this is the same as asm.strip

Reply:
dont use more than one dot in the eval key names if there's no more than one. so followcf.hide is not a good name.

maybe we can do asm.hide.branch and asm.hide.strstr